### PR TITLE
Add glossary entry for "Special Page"

### PR DESCRIPTION
**Pull request by Silvan-WMDE**
Bug: T272706

### CODE REVIEW

File: systems/overview/12-Glossary.md [hunk 1]
```diff
@@ -51,3 +51,7 @@ Wikitext, also known as Wiki markup or Wikicode, consists of the syntax and keyw
 ## [MediaWiki Core](https://www.mediawiki.org/wiki/Core)
 
 MediaWiki Core refers to all files that are part of the main MediaWiki package.
+
+## [Special Page](https://www.mediawiki.org/wiki/Manual:Special_pages)
```
Comment (addshore) [inline systems/overview/12-Glossary.md:55]: Should we just link to or at least used the same wording as on https://www.mediawiki.org/wiki/Special_pages ?

File: systems/WikibaseRepo/05-Building_Block_View.md [hunk 2]
```diff
@@ -79,9 +79,9 @@ TBA
...
+| SpecialWikibasePage                         | base class          | Base class for Wikibase [Special Pages](../overview/12-Glossary.md#special-page)                                                                                                                             |
 | EntityIdParser                              | Handling user input | Interface for objects that can parse strings into EntityIds                                                                                                                                                  |
-| TokenCheckInteractor                        | Handling user input | Checks a submitted edit token                                                                                                                                                                                 |
+| TokenCheckInteractor                        | Handling user input | Checks a submitted edit token                                                                                                                                                                                |
```
Comment (addshore) [inline systems/WikibaseRepo/05-Building_Block_View.md:84]: Not sure what happened in this line?
I don't see any change?

Comment (Silvan-WMDE) [reply systems/WikibaseRepo/05-Building_Block_View.md:84]: I think the md linter added this: I think it deleted a single space. I kept fighting against it, then gave up at some point.

File: systems/overview/12-Glossary.md (hunk 1 above)
Comment (Silvan-WMDE) [reply systems/overview/12-Glossary.md:55]: I don't think a simple link would fit the purpose of a Glossary. I slightly modified the wording from https://www.mediawiki.org/wiki/Special_pages in an attempt to shorten it, but am happy to use exactly that, when you think we should stick to the original.

**Review by addshore — APPROVED** [review]

**Review by tzhelyazkova — APPROVED** [review]

**Pull request merged**